.github/workflows/ecspresso-feature-branch.yml

```diff
@@ -0,0 +1,136 @@
+name: |-
+  Features branch (Pull request) workflow 
+  
+  Build, test Docker image, deploy it with Spacelift to QA environments depends of the PR labels  
+  
+  ### Usage 
+  
+  Create in your repo  __`.github/workflows/feature.yaml`__
+  
+  ```yaml
+    name: Feature Branch
+    on:
+      pull_request:
+        branches: [ 'master' ]
+        types: [opened, synchronize, reopened, closed, labeled, unlabeled]
+    
+    permissions:
+      pull-requests: write
+      deployments: write
+      id-token: write
+      contents: read
+    
+    jobs:
+      do:
+        uses: cloudposse/github-actions-workflows-docker-ecr-spacelift/.github/workflows/feature-branch.yml@main
```

Comment:
and dont forget to change workflow path from `cloudposse/github-actions-workflows-docker-ecr-spacelift/.github/workflows` to `./.github/workflows`